nextflow.config

```diff
@@ -15,6 +15,15 @@ params {
   outdir = './results'
   igenomes_base = 's3://ngi-igenomes/igenomes/'
   clusterOptions = false
+  
+  // Pipeline version
+  version = '1.4dev'
```

Comment:
This is good, but the variable name in `main.nf` now needs to be updated to `params.version` instead of just `version`.

Reply:
Also, small thing but if these two lines could be at the top that would be great, as they're the only parts of this config that need to be regularly updated.